### PR TITLE
Merge loops

### PR DESCRIPTION
**Pull request by erdinctaskin**
Merge 2 loops into single loop

### CODE REVIEW

**Comment by nfrankel** [comment]
run-lab-run

File: hazelcast/src/main/java/com/hazelcast/internal/util/UuidUtil.java [hunk 1]
```diff
@@ -101,10 +101,9 @@ private static UUID getUUID(Random random) {
         assert data.length == 16 : "data must be 16 bytes in length";
         for (int i = 0; i < 8; i++) {
             mostSigBits = (mostSigBits << 8) | (data[i] & 0xff);
+            leastSigBits = (leastSigBits << 8) | (data[i + 8] & 0xff);
```
Comment (ivanthescientist) [inline hazelcast/src/main/java/com/hazelcast/internal/util/UuidUtil.java:104]: Is there a test/benchmark that demonstrates that this is faster than 2 loops actually? Ideally in this scenario compiler/JIT should unroll these 2 loops into series of instructions (or actually ideally it should be just two `movq`s really or even a single `movdqa` ). I did a very dumb test, executed the new and old impls of this method 1000000 times and measured total time, long story its within margin of error on my machine, its probably flawed though.

Imo the readability is better with the original, so I'd say unless this provides a massive perf boost, we should keep the original.

**Comment by Holmistr** [comment]
Hey @erdinctaskin @ivanthescientist , can we proceed with this PR? Any comments?

**Comment by Holmistr** [comment]
Hey @erdinctaskin @ivanthescientist , any comments? Otherwise I'll close this PR for inactivity.

**Pull request closed without merging**